pkg/sessionctx/variable/sysvar.go

```diff
@@ -1017,6 +1017,14 @@ var defaultSysVars = []*SysVar{
 			return normalizedValue, nil
 		},
 	},
+	{Scope: vardef.ScopeGlobal, Name: vardef.MaxUserConnections, Value: strconv.FormatUint(vardef.DefMaxUserConnections, 10), Type: vardef.TypeUnsigned, MinValue: 0, MaxValue: 100000,
```

Comment:
max value should be 4294967295. And seems it has a session level scope 🤔 

https://dev.mysql.com/doc/refman/8.4/en/server-system-variables.html#sysvar_max_user_connections

Reply:
Thie requirement has been refined by PM. 
requirement details in https://pingcap.feishu.cn/wiki/ICyGwOmDAiNmsxkRwiacyz4Ln3g.

Reply:
don't paste internal doc here, you can chat offline

Reply:
> max value should be 4294967295. And seems it has a session level scope 🤔

the session one is only a read-only var

> It also has a read-only session value that indicates the effective simultaneous-connection limit that applies to the account associated with the current session.